structurizr-plantuml/src/com/structurizr/io/plantuml/C4PlantUMLWriter.java

```diff
@@ -163,6 +163,9 @@ void doWrite(View view, Component element, Writer writer, String prefix, String
 		@Override
 		void doWrite(View view, SoftwareSystemInstance element, Writer writer, String prefix, String id, String separator)
 				throws IOException {
+			if (view.getElementView(element) == null) {

```

Comment:
I realise I could've used `isElementInView()` 🤦‍♂️ 

Oh well. Thank you for merging!

Reply:
No problem, thanks for the fix. I'd actually forgotten `isElementInView()` existed already, as I went in to create it in the core library after I merged the PR! 😄